wiki/fr/fr-index.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: wikioverview
+title: "Vue d'ensemble"
+lang: "fr"
+permalink: "/wiki/"
+---
+# Vue d'ensemble du Wiki
```

Comment:
"Vue d'ensemble du Wiki" -> "Vue d'ensemble des pages"

(The use of the word "wiki" reduces understanding to people who know what a "wiki" is. In addition, the original English version text uses the word "page", not "wiki".)

Reply:
« Aperçu » sounds better for me.

But in the context it is the map or the list of the pages.
It could be « carte du site ».

This can be changed later...